### PR TITLE
Update Readme. Added PersonaServer project

### PR DESCRIPTION
**Pull request by babaktaremi**
Added PersonaServer project to readme file

### CODE REVIEW

**Comment by babaktaremi** [comment]
No approval?😔

**Comment by kevinchalet** [comment]
> No approval?😔

It's Sunday... 😞

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ This repository contains samples demonstrating **how to use [OpenIddict](https:/
 **Looking for additional samples to help you get started with OpenIddict?** Don't miss these interesting samples maintained by the community:
 
   - **[Angular and Blazor samples](https://github.com/damienbod/AspNetCoreOpeniddict)** by [Damien Bowden](https://github.com/damienbod)
+ - **[PersonaServer .An authentication server that implements OpenID Connect (OIDC) and OAuth 2.0 standards using OpenIddict. With user management implementation using ASP NET Core Identity](https://github.com/babaktaremi/PersonaServer)** by [Babak Taremi](https://github.com/babaktaremi)
```
Comment (kevinchalet) [inline README.md:29]: The spacing is weird and the description is likely too long.

```suggestion
  - **[PersonaServer: an authentication server that implements OpenID Connect and OAuth 2.0 using OpenIddict and ASP.NET Core Identity](https://github.com/babaktaremi/PersonaServer)** by [Babak Taremi](https://github.com/babaktaremi)
```

**Comment by kevinchalet** [comment]
Thanks for your PR.

Before merging it, is PersonaServer already working or is it still a WIP?

**Comment by babaktaremi** [comment]
> Thanks for your PR.
> 
> Before merging it, is PersonaServer already working or is it still a WIP?

Yep it's a WIP and it's not "Ready to deploy " but it provides a good starting point. I'm working on many extra features such as Admin-Panel , User Panel, One time authorizations, 2FA authentication and Integration with other external login providers such as google, facebook etc

**Comment by kevinchalet** [comment]
Good, thanks! In this case, I suggest we wait a bit before merging this PR 😃

**Comment by kevinchalet** [comment]
It looks like the project is no longer under active development. Closing for now 😃

**Pull request closed without merging**